pipelines/exposureQualityCore.yaml

```diff
@@ -0,0 +1,13 @@
+description: |
+  Tier 1 plots and metrics to assess exposure quality
+tasks:
+  analyzeAmpOffsetTaskCore:
+    class: lsst.analysis.tools.tasks.IsrAmpOffsetMetricsAnalysisTask
```

Comment:
Should the task be named metrics since tools within it might contain plots?

Reply:
Task renamed to `IsrMetadataExposureDetectorAnalysisTask` following review.